.github/workflows/checks.yml

```diff
@@ -75,13 +75,9 @@ jobs:
           version: "23.2"
           repo-token: ${{ secrets.GITHUB_TOKEN }}
       - name: Toolchain
-        uses: actions-rs/toolchain@v1
+        uses: dtolnay/rust-toolchain@nightly
```

Comment:
We need both stable and nightly

Reply:
For lints? that's not actually the case. See my other comment